test/diff/attr_set/out-pure.nix

```
@@ -370,10 +334,7 @@
       # Sans) than that horror.  But we do need the Adobe fonts for some
       # old non-fontconfig applications.  (Possibly this could be done
       # better using a fontconfig rule.)
-      [
-        pkgs.xorg.fontadobe100dpi
-        pkgs.xorg.fontadobe75dpi
-      ];
+      [ pkgs.xorg.fontadobe100dpi pkgs.xorg.fontadobe75dpi ];
```

This is a case where the expanded formatting is preferred:

```suggestion
      [
        pkgs.xorg.fontadobe100dpi
        pkgs.xorg.fontadobe75dpi
      ];
```